packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -2140,12 +2139,10 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
             ),
           ),
           child: SafeArea(
-            child: ButtonTheme.bar(
-              child: SafeArea(
-                top: false,
-                child: ButtonBar(
-                  children: widget.persistentFooterButtons,
-                ),
+            child: SafeArea(
```

Comment:
Hmm I know you didn't add this but the double safe area seems a bit strange to me, is this necessary?

Reply:
yeah, not sure what was up with that. I was just mechanically removing `ButtonTheme.bar` but looking at it now that is really odd. I removed it and it doesn't seem to break anything.